src/components/SearchProducts/SearchProducts.tsx

```diff
@@ -0,0 +1,23 @@
+import { TouchableOpacity } from 'react-native';
+import { SearchProductCard } from './SearchProductCard';
+import { Product } from '../ProductSlide';
+
+interface Props {
+    product: Product,
+    position: number
+}
+
+const SearchProducts: React.FC<Props> = ({ product, position }) => {
+
+    const handleButtonClick = () => {
+        console.log(product.title)
+    }
+
+    return (
+        <TouchableOpacity onPress={() => handleButtonClick()}>
+            <SearchProductCard product={product} position={position}/>
+        </TouchableOpacity>
+    );
+}
+
+export default SearchProducts;
```

Comment:
Also, I created a card called `ProductCart`, why you don't use it here?

Reply:
I didn't used it here because I had to do some small changes into the component. For example, changed the width, fixed a height and created a conditional to set a margin to become similar as the page in figma. However, I will try to use only the product card